src/core/messaging/MultiAdapter.sol

```diff
@@ -200,28 +200,38 @@ contract MultiAdapter is Auth, IMultiAdapter {
         }
     }
 
+    /// @dev Internal helper to get the first adapter's details for a pool, handling empty cases
+    function _getFirstAdapterDetails(uint16 centrifugeId, PoolId poolId)
```

Comment:
Nit: would put internal method below the last external view method

Reply:
That was also my intuition but we have an internal view above this one already.

Reply:
Yeah but the internal view method (`_poolAdapterDetails`) is below the external method (`poolAdapters`) that uses it. Here similarly, `_getFirstAdapterDetails` can be below the external methods that use it.

Or we can introduce a new `Internal methods`/`Helpers` section below this

Reply:
I prefer leaving internal methods just below the method that calls them. Otherwise it's more difficult to read the contract